src/library/dlgtagfetcher.cpp

```diff
@@ -250,45 +308,67 @@ void DlgTagFetcher::fetchTagFinished(
 }
 
 void DlgTagFetcher::slotNetworkResult(
-        int httpError, const QString& app, const QString& message, int code) {
+        int httpError,
+        const QString& app,
+        const QString& message,
+        int code) {
     m_networkResult = httpError == 0 ? NetworkResult::UnknownError : NetworkResult::HttpError;
     m_data.m_pending = false;
-    QString strError = tr("HTTP Status: %1");
-    QString strCode = tr("Code: %1");
-    httpStatus->setText(strError.arg(httpError) + "\n" + strCode.arg(code) + "\n" + message);
-    QString unknownError = tr("Mixxx can't connect to %1 for an unknown reason.");
-    cantConnectMessage->setText(unknownError.arg(app));
-    QString cantConnect = tr("Mixxx can't connect to %1.");
-    cantConnectHttp->setText(cantConnect.arg(app));
+
+    if (m_networkResult == NetworkResult::UnknownError) {
```

Comment:
I think we can now dispose the whole if branch.

Reply:
We can probably dispose the whole m_networkResult

Reply:
Done with the last commit 👍